ckanext/switzerland/dcat/harvesters.py

```diff
@@ -90,3 +102,9 @@ def _gen_new_name(self, title):
             return super(SwissDCATRDFHarvester, self)._gen_new_name(title['de'])  # noqa
         except TypeError:
             return super(SwissDCATRDFHarvester, self)._gen_new_name(title)  # noqa
+
+    def before_create(self, harvest_object, dataset_dict, temp_dict):
+        if self.config.get('excluded-dataset-identifiers'):
+            for excluded_dataset_identifier in self.config.get('excluded-dataset-identifiers'):  # noqa
+                if excluded_dataset_identifier == dataset_dict.get('identifier'):  # noqa
+                    dataset_dict = None
```

Comment:
In order to have an effect on the dataset_dict that is passed into the method, you must change it. So simply assigning `None` to the parameter will change nothing, instead you must use `dataset_dict.clear()` to "empty" the original dict.